### PR TITLE
User EDITS event spec (sorry for branch name)

### PR DESCRIPTION
**Pull request by mhgoldman**
Do you think this is enough coverage? Trying to walk the line between totally duplicating the tests on create while testing enough stuff.

### CODE REVIEW

File: spec/features/user_edits_event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    fill_in_date_time 'End time', with: ''
+
+    click_button 'Update Event'
+
+    [ "Please review the problems below",
+      "Name can't be blank",
+      "Start time must be supplied in the format MM/DD/YYYY HH:MM AM/PM",
+      "End time must be supplied in the format MM/DD/YYYY HH:MM AM/PM"
+    ].each do |msg|
+      expect(page).to have_form_alert(msg)
```
Comment (eliotsykes) [inline spec/features/user_edits_event_spec.rb]: Consider making `have_form_alert`:
- accept 1+ message arguments with splat
- aliasing it so its also called `have_form_alerts`

Making the following all possible:

```ruby
expect(page).to have_form_alert("Some message")

expect(page).to have_form_alerts("Msg 1", "Msg 2", ...)
```

File: spec/features/user_edits_event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    fill_in 'Name', with: 'Some other name'
+    fill_in_date_time 'Start time', with: '03/04/2055 6:15 PM'
+    fill_in_date_time 'End time', with: '03/09/2055 8:30 AM'
+    select "(event's local time zone)", from: 'event_times_indicated_in_tz'
+    
+    click_button 'Update Event'
+
+    expect(page).to have_flash('Event updated successfully')
+
```
Comment (eliotsykes) [inline spec/features/user_edits_event_spec.rb]: Consider adding an expectation after the flash expectation (line 21) to check that a successful update takes the user to the correct page.

**Comment by eliotsykes** [comment]
> Do you think this is enough coverage? Trying to walk the line between totally duplicating the tests on create while testing enough stuff.

Only thing that I could see missing was checking the user is taken to the correct page on a successful update.

**Comment by mhgoldman** [comment]
Actually, something that occurred to me - was there a problem you had with
starting to fill_in a form without first checking the page for content? I
don't think I did that everywhere, but I can.

**Comment by eliotsykes** [comment]
> Actually, something that occurred to me - was there a problem you had with starting to fill_in a form without first checking the page for content? I don't think I did that everywhere, but I can.

That might end up being a problem. The problem I hit was to do with `click_link` happening twice in succession without waiting for the content to update in-between clicks.

**Pull request merged**